1.2/spec.md

```diff
@@ -975,35 +1002,28 @@ Depending on the particular HTML markup used in the document, the document is
 suitable for different kinds of processing and use. The formats have the
 following intents:
 
-## `html_none`
-
-Straightforward equivalent of Goodoc or [[XDOC]]
-
-## `html_ocr`
-
-Straightforward recording of commercial OCR system output
-
-## `html_absolute`
-
-Target format for services like Google's View as HTML
-
-## `html_xytable`
-
-Target format for layout-preserving on-screen document viewing
-
-## `html_simpl`
-
-Target format for convenient on-line viewing and intermediate format for indexing
+  : <a>html_none</a> (see [[#format-none]])
+  :: Straightforward equivalent of Goodoc or [[XDOC]]
+  : <a>html_simple</a>
+  :: Target format for convenient on-line viewing and intermediate format for indexing
+  : <a>html_xytable_absolute</a>, <a>html_xytable_relative</a>
```

Comment:
What happened here? This looks like some merge of "html_xytable" and "html_absolute"...